lib/WUI/espif.cpp

```diff
@@ -443,7 +444,23 @@ static err_t low_level_output(struct netif *netif, struct pbuf *p) {
     return ERR_OK;
 }
 
+static void force_down() {
+    struct netif *iface = active_esp_netif; // Atomic load
+    assert(iface != nullptr);               // Already initialized
+    process_link_change(false, iface);
```

Comment:
Assert should be (also) in `netif_set_link_up` and `netif_set_link_down`. Then there's no need for `iface` here.

Reply:
OK for adding it to the `process_link_change`, I've added it there. But:

* `netif_set_link_*` are 3rd-party ones from LwIP, they are not going to be changed.
* I want it here not only to catch bugs (that's nice), but as a documentation. I'm claiming this function is called only after everything has been initialized.